lib/streamlit/elements/widgets/button.py

```diff
@@ -421,6 +424,30 @@ def link_button(
             use_container_width=use_container_width,
         )
 
+    @gather_metrics("page_link")
+    def page_link(
+        self,
+        page: str,
+        *,
+        label: Optional[str] = None,
```

Comment:
nit: I think the new annotations format is slightly preferred here:

`label: str | None = None,`

You just need to add `from __future__ import annotations` to the top.